src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -43,6 +39,23 @@ def __init__(self, dataManager, logger, rucioAccount, rucioClient):
         self.rucioClient = rucioClient
         self.report = MSPileupReport()
 
+    def pileupSizeTask(self):
+        """
+        Execute pileup size update task
+        """
+        # get pileup sizes and update them in DB
+        spec = {}
```

Comment:
I don't have any great suggestions, but in general we should not fetch the pileup size in every cycle. They can change during their lifetime - as data gets lost/invalidated - but that should be fairly uncommon.

I was considering to use a spec like `pileupSize != 0`, such that we only update the pileup size for containers that have not been updated. However, in this mode we would update it only once in its lifetime, which might miss the use case mentioned above...